utils/utils.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+# NOTE: The strtobool function should be reimplemented since the distutils module is deprecated
+# See details in PEP 632 – https://peps.python.org/pep-0632/#migration-advice
+def strtobool(val: str) -> bool:
```

Comment:
FYI, there's a function in attrs that's almost exactly like this one: [`attrs.converters.to_bool`](https://www.attrs.org/en/stable/api.html#attrs.converters.to_bool).